repose-aggregator/components/filters/header-translation-filter/src/main/scala/org/openrepose/filters/headertranslation/HeaderTranslationFilter.scala

```diff
@@ -36,17 +36,17 @@ import scala.collection.JavaConversions._
 class HeaderTranslationFilter @Inject()(configurationService: ConfigurationService)
   extends Filter with UpdateListener[HeaderTranslationType] with LazyLogging {
 
-  private final val DEFAULT_CONFIG = "header-translation.cfg.xml"
+  import HeaderTranslationFilter._
```

Comment:
I know we've done this in quite a few places, but I wonder if it's worth breaking constants out into an Object. Specifically for filters, since they are singleton Spring beans anyway. It may be easier to see the constants inline, and there is likely a very, _very_ small overhead that comes with defining a class and an object.

It doesn't bother me that much, but I figured I'd write out my musings anyway.